service/service.go

```diff
@@ -184,7 +184,7 @@ func New(config Config) (*Service, error) {
 // Boot starts top level service implementation.
 func (s *Service) Boot(ctx context.Context) {
 	s.bootOnce.Do(func() {
-		go s.operatorCollector.Boot(ctx)
+		go s.operatorCollector.Boot(ctx) // nolint:errcheck
```

Comment:
There's a sample fix for this errcheck in kvm-operator if you like

Reply:
Nice thanks I've updated to use that.